### DISCUSSION TITLE
v6.1.0

### DISCUSSION THREAD

**Discussion by tay1orjones** [IBM/plex#468] [General]
### Added
- glyphs /acaron /acaron.alt01 /icaron /ocaron /ucaron /udieresismacron /udieresisacute /udieresisgrave /udieresiscaron /Acaron /Icaron /Ocaron /Ucaron /Udieresismacron /Udieresisacute /Udieresisgrave /Udieresiscaron to support pinyin
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
- glyph /longs
  referenced in [issue 158](https://github.com/IBM/plex/issues/158)
- glyph /divisionslash
- old-style numerals
  referenced in [issue 418](https://github.com/IBM/type/issues/418)
- Cyrillic support for Abkhaz language
  referenced in [issue 355](https://github.com/IBM/type/issues/355)
- support for pre-1918 Cyrillic
  referenced in [issue 158](https://github.com/IBM/type/issues/158)
- U+2236 RATIO
  referenced in [issue 199](https://github.com/IBM/plex/issues/199)
- U+2219 BULLET OPERATOR 
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
- U+2010-U+2015 NON-BREAKING HYPHEN, DASHES, etc.
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 317](https://github.com/IBM/plex/issues/317)
- U+2028 LINE SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+2029 PARAGRAPH SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+202F NARROW NO-BREAK SPACE
  referenced in [issue 282](https://github.com/IBM/plex/issues/282)
- U+2008 PUNCTUATION SPACE
- U+200A HAIR SPACE
  referenced in [issue 361](https://github.com/IBM/plex/issues/361)
- U+200B ZERO WIDTH SPACE
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 318](https://github.com/IBM/plex/issues/318)

### Fixed
- FamilyClass value in OS/2 table
  referenced in [issue 330](https://github.com/IBM/plex/issues/330)
- PostScript BlueScale values

### Changed
- Improved shape for Cyrillic letter Fita (U+0473, U+0472)
  referenced in [issue 158](https://github.com/IBM/type/issues/158)
- replaced commaaccent with cedilla in glyphs /Tcedilla /tcedilla
  referenced in [issue 251](https://github.com/IBM/plex/issues/251)
- disabled bit 38 (Mathematical Operators) from OS/2 UnicodeRanges

### Added
- old-style numerals
  referenced in [issue 418](https://github.com/IBM/type/issues/418)
- U+2236 RATIO
  referenced in [issue 199](https://github.com/IBM/plex/issues/199)
- U+2219 BULLET OPERATOR 
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
- U+2010-U+2015 NON-BREAKING HYPHEN, DASHES, etc.
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 317](https://github.com/IBM/plex/issues/317)
- U+2028 LINE SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+2029 PARAGRAPH SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+202F NARROW NO-BREAK SPACE
  referenced in [issue 282](https://github.com/IBM/plex/issues/282)
- U+2008 PUNCTUATION SPACE
- U+200A HAIR SPACE
  referenced in [issue 361](https://github.com/IBM/plex/issues/361)
- U+200B ZERO WIDTH SPACE
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 318](https://github.com/IBM/plex/issues/318)

### Fixed
- FamilyClass value in OS/2 table
  referenced in [issue 330](https://github.com/IBM/plex/issues/330)
- PostScript BlueScale values

### Added
- old-style numerals
  referenced in [issue 418](https://github.com/IBM/type/issues/418)
- Cyrillic support for Abkhaz language
  referenced in [issue 355](https://github.com/IBM/type/issues/355)
- support for pre-1918 Cyrillic
  referenced in [issue 158](https://github.com/IBM/type/issues/158)
- U+2236 RATIO
  referenced in [issue 199](https://github.com/IBM/plex/issues/199)
- U+2219 BULLET OPERATOR 
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
- U+2010-U+2015 NON-BREAKING HYPHEN, DASHES, etc.
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 317](https://github.com/IBM/plex/issues/317)
- U+2028 LINE SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+2029 PARAGRAPH SEPERATOR
  referenced in [issue 52](https://github.com/IBM/type/issues/52)
- U+202F NARROW NO-BREAK SPACE
  referenced in [issue 282](https://github.com/IBM/plex/issues/282)
- U+2008 PUNCTUATION SPACE
- U+200A HAIR SPACE
  referenced in [issue 361](https://github.com/IBM/plex/issues/361)
- U+200B ZERO WIDTH SPACE
  referenced in [issue 84](https://github.com/IBM/plex/issues/84)
  referenced in [issue 318](https://github.com/IBM/plex/issues/318)

### Fixed
- FamilyClass value in OS/2 table
  referenced in [issue 330](https://github.com/IBM/plex/issues/330)
- PostScript BlueScale values

### Changed
- Improved shape for Cyrillic letter Fita (U+0473, U+0472)
  referenced in [issue 158](https://github.com/IBM/type/issues/158)

<hr /><em>This discussion was created from the release <a href='https://github.com/IBM/plex/releases/tag/v6.1.0'>v6.1.0</a>.</em>

**Comment by codeman38** [comment]
Something seems to have gone extremely wrong with the style naming in the TTF version of Plex Serif in this release - filed #469 about this.

**Reply by tay1orjones** [reply]
https://github.com/IBM/plex/issues/469#issuecomment-1218204237

> We'll work on getting a fix out. For now if you need to use TTF you'll need to revert back to v6.0.2.

**Reply by tay1orjones** [reply]
[v6.1.1](https://github.com/IBM/plex/releases/tag/v6.1.1) is out and fixes this issue

**Comment by rfpng** [comment]
Sorry, I'm confused about the version numbers. 
I just downloaded the newest TrueType and OpenType fonts and installed the IBMPlex**Mono** TTF fonts (in Windows 10). The Version number for these fonts shows **Version 2.3**. 
The IBMPlex**Sans** fonts, however, show **Version 3.004**, the IBMPlex**Serif** fonts show **Version 2.008**, etc
Nowhere can I find the Version number 6.1.0.
Is this the way it should be or am I doing something wrong?

**Reply by tay1orjones** [reply]
@rfpng Sorry for the confusion - v6.1.0 contains multiple different families that each have their own respective version numbers. The collection of all these together comprise the v6.1.0 release that is published to `@ibm/plex`. Not every family is updated in every release so the discrete family version numbers will never align to one another, nor the version listed here and published to `@ibm/plex`.

The full list of everything and the discrete version numbers can be found on the [releases page](https://github.com/IBM/plex/releases) where all the changelogs are published.

**Comment by nashpatty** [comment]
Are old style figures going to be added to mono too? I can see they're added to sans.

**Reply by tay1orjones** [reply]
cc @BoldMonday

**Reply by BoldMonday** [reply]
An update for Plex Mono is underway which will include hanging numerals.
